lib/protocol/http/header/etags.rb

```diff
@@ -8,15 +8,32 @@
 module Protocol
 	module HTTP
 		module Header
-			# This implementation is not strictly correct according to the RFC-specified format.
 			class ETags < Split
 				def wildcard?
 					self.include?('*')
 				end
 				
+				# This implementation is not strictly correct according to the RFC-specified format.
 				def match?(etag)
 					wildcard? || self.include?(etag)
 				end
+				
+				# Useful with If-Match
+				def strong_match?(etag)
+					wildcard? || self.reject{|t| weak_tag?(t)}.include?(etag)
```

Comment:
IIUC, `reject` is just removing any tags that start with `W/`. Then on that set, we check if it includes `etag`. Why not just reject if `etag` is weak? This avoids scanning the list twice?

Reply:
Good call. Done!